<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui_main_window</class>
 <widget class="QDialog" name="ui_main_window">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>849</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">QuickOSM</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QListWidget" name="menu_widget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>153</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{
	background-color: rgb(69, 69, 69, 220);
	outline: 0;
}
QListWidget::item {
	color: white;
	padding: 3px;
}
QListWidget::item::selected {
	color: black;
	background-color:palette(Window);
    padding-right: 0px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Mesh creation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Boundaries creation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mesh interpolation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>About</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="content_widget">
     <item>
      <widget class="QgsMessageBar" name="message_bar">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="main_scroll">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="main_scroll_inside">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>827</width>
          <height>780</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>783</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stacked_panels_widget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="mesh_creation">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="1">
              <widget class="QgsMapLayerComboBox" name="combo_countour_layer_mp"/>
             </item>
             <item row="3" column="1">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="countour_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Contour</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="iles_label">
               <property name="text">
                <string>Iles</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="LayerSelectionCreation" name="combo_iles_layer" native="true"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="boundaries_creation">
            <layout class="QVBoxLayout" name="verticalLayout_9"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="progress_box">
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="progress_text">
         <property name="text">
          <string notr="true">progress text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QFrame</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayerSelectionCreation</class>
   <extends>QWidget</extends>
   <header>salome_hydro_qgis.ui.layer_selection_creation</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>main_scroll</tabstop>
  <tabstop>menu_widget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
